src/code/z_scene.c

```diff
@@ -589,13 +589,13 @@ s32 Scene_ProcessHeader(PlayState* play, SceneCmd* header) {
 /**
  * Creates an entrance index from the scene index, spawn index, and scene setup.
  */
-u16 Entrance_CreateIndex(s32 sceneIndex, s32 spawnIndex, s32 sceneSetup) {
-    return (((sceneIndex << 9) | (spawnIndex << 4)) | sceneSetup) & 0xFFFF;
+u16 Entrance_CreateIndex(s32 scene, s32 spawn, s32 layer) {
+    return (scene << 9) | (spawn << 4) | layer;
 }
```

Comment:
```suggestion
u16 Entrance_CreateEntry(s32 scene, s32 spawn, s32 layer) {
    return (scene << 9) | (spawn << 4) | layer;
}
```
I wonder if this should become something like this, now that it's not `entranceIndex`. Same thing for `Entrance_CreateIndexFromSpawn`

Reply:
Yeah, that makes sense.